cypress/support/index.js

```diff
@@ -51,3 +51,20 @@ Cypress.on('scrolled', $el => {
     inline: 'center',
   });
 });
+
+// This adds a delay between cypress commands for easier debugging
+// Ref: https://github.com/cypress-io/cypress/issues/249
+//
+// const COMMAND_DELAY = 1000;
+//
+// for (const command of ['visit', 'click', 'trigger', 'type', 'clear', 'reload', 'contains']) {
+//   Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+//     const origVal = originalFn(...args);
+//
+//     return new Promise((resolve) => {
+//       setTimeout(() => {
+//         resolve(origVal);
+//       }, COMMAND_DELAY);
+//     });
+//   });
+// }
```

Comment:
I know it's commented now,but in which test have you tried it? how useful it was comparing with a classic timeout?

Reply:
This adds a delay for all the things cypress does, that means clicking, typing, testing for a get/should/contains, etc.
It's different than using the normal `cy.wait()`. I have tested it in one of the tests which were failing, where i wanted to live-track what was going on, and it works ok :)